mindmeld/components/request.py

```diff
@@ -150,6 +209,7 @@ class FrozenParams(Params):
     target_dialogue_state = attr.ib(default=None)
     time_zone = attr.ib(default=None)
     timestamp = attr.ib(default=0)
+    language = attr.ib(default='en')
```

Comment:
This is inconsistent with the `EN` we have as the default above on line 138

Reply:
good point, resolved